quiet_logistics_endpoint.rb

```diff
@@ -218,6 +218,25 @@ class QuietLogisticsEndpoint < EndpointBase::Sinatra::Base
     result code, message
   end
 
+  post '/get_errors' do
```

Comment:
Is this plurality accurate? Will we be returning errors or just 1 error?

Reply:
Ahh, it would be only one technically but this endpoint would be set up as a trigger so it should get multiple errors in all.